core/src/main/scala/filodb.core/memstore/TimeSeriesMemStore.scala

```diff
@@ -158,7 +158,11 @@ extends MemStore with StrictLogging {
           startOffsetValidated = true
         }
         shard.ingest(r)
-      }.collect {
+      }
+        // Nothing to read from source, blocking indefinitely. In such cases, 60sec timeout causes it
+        // to return endOffset, making recovery complete and to start normal ingestion.
+        .timeoutOnSlowUpstreamTo(timeout, Observable.now(endOffset))
```

Comment:
Could we dedent two spaces on these two lines so that they line up with the previous closing brace, and so that `.collect {` appears one level up from the case?

Reply:
@velvia fixed the indentation